driven/storage/adapter.go

```diff
@@ -603,6 +603,17 @@ func (sa *Adapter) SaveUser(user *model.User) error {
 	return nil
 }
 
+func (sa *Adapter) GetCovid19Configs() ([]model.COVID19Config, error) {
```

Comment:
We use the _lint_ tool to flag programming errors, bugs, stylistic errors, and suspicious constructs. The command is "make lint".
When I run it then it gives the following errors:
```
▶ running golint…
.../core/administration.go:41:1: exported method Application.GetCovid19Configs should have comment or be unexported
.../driven/storage/adapter.go:606:1: exported method Adapter.GetCovid19Configs should have comment or be unexported
.../driver/web/rest/adminapis.go:59:1: exported method AdminApisHandler.GetCovid19Configs should have comment or be unexported
Found 3 lint suggestions; failing.
make: *** [lint] Error 1
```